packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -242,6 +243,7 @@ contract L2Genesis is Deployer {
         setSchemaRegistry(); // 20
         setEAS(); // 21
         setGovernanceToken(); // 42: OP (not behind a proxy)
+        setGovernanceDelegation(); // 43: OP (not behind a proxy)
```

Comment:
Can we put `setGovernanceDelegation` behind the `useInterop()` flag? Since it will be having interop features added soon